### DISCUSSION TITLE
2.40.4 / 2022-11-29

### DISCUSSION THREAD

**Discussion by roidelapluie** [prometheus/prometheus#11637] [Announcements]
* [SECURITY] Fix basic authentication bypass vulnerability (CVE-2022-46146). GHSA-4v48-4q5m-8vx4

<hr /><em>This discussion was created from the release <a href='https://github.com/prometheus/prometheus/releases/tag/v2.40.4'>2.40.4 / 2022-11-29</a>.</em>

**Comment by yasharne** [comment]
Where can I find information about `CVE-2022-46146`? I didn't find anything by searching on the google or https://nvd.nist.gov/

**Comment by roidelapluie** [comment]
See https://groups.google.com/g/prometheus-announce/c/qVv_BpMJdnM and https://www.openwall.com/lists/oss-security/2022/11/29/1
